tests/tnt/tnt_test.go

```diff
@@ -14,6 +14,9 @@ const (
 
 	envNreplicasetsKey = "NREPLICASETS"
 	envStartPortKey    = "START_PORT"
+
+	defaultTntUser     = "storage"
```

Comment:
To be honest, this is the first time I’ve seen such passwords. usually examples from tarantool came without a password, take the same https://github.com/tarantool/vshard/tree/master/example

Reply:
The lua vshard example is the exactly template I have based on. They use the same user and password: https://github.com/tarantool/vshard/blob/dfa2cc8a2aff221d5f421298851a9a229b2e0434/example/localcfg.lua#L6